packages/types/lib/driver.ts

```diff
@@ -189,7 +189,12 @@ export interface MultiSessionData<C extends Constraints = Constraints> {
 }
 
 /**
- * Data returned by {@linkcode ISessionCommands.getSession}.
+ * Data returned by {@linkcode ISessionHandler.getAppiumSessionCapabilities}.
+ */
+export interface AppiumSessionCapabilities {}
```

Comment:
not sure if we need a new type for that. Usually each driver already has a type for its caps

Reply:
do we generally know how the output of this API should look like?

Reply:
I think we haven't decided yet, so I haven't specified details yet. This PR just replaced better error and response from the umbrella driver

Reply:
lets remove this typedef for now then and return `any` until we have a clear decision